pupa/models.py

```diff
@@ -41,3 +41,10 @@ class ImportObjects(models.Model):
     noop_count = models.PositiveIntegerField()
     start_time = models.DateTimeField()
     end_time = models.DateTimeField()
+
+
+class Measures(models.Model):
```

Comment:
This might get confusing (since there is a type of Legislation that folks call Measures) -- perhaps `Measurement` or `DataQualityChecks` or something?